client/my-sites/checkout/checkout/ideal-payment-box.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PaymentBox from './payment-box';
+import { localize, translate } from 'i18n-calypso';
+
+class IdealPaymentBox extends PureComponent {
+	handleToggle = ( event ) => {
+		event.preventDefault();
+		this.props.onToggle( 'credit-card' );
+	};
+
+	render() {
+		return (
+			<PaymentBox
+				classSet="ideal-payment-box"
+				title={ translate( 'Secure Payment with iDEAL' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'Secure Payment with PayPal' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/payment-ideal)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).